### ISSUE TITLE
Cannot use {% url path %} in blog posts

### ISSUE THREAD

**Issue by blattms** [eudicots/Cactus#234]
Using `{% url path %}` in blog posts results in the following error:
```
*** Error while building
'__CACTUS_SITE__'
Traceback (most recent call last):
...
File "/usr/local/lib/python2.7/dist-packages/Cactus-3.3.2-py2.7.egg/cactus/template_tags.py", line 52, in url
    site = context['__CACTUS_SITE__']
  File "/usr/local/lib/python2.7/dist-packages/Django-1.6.11-py2.7.egg/django/template/context.py", line 56, in __getitem__
    raise KeyError(key)
KeyError: '__CACTUS_SITE__'
```

**Comment by dwightgunning** [comment]
Try modifiying the blog.py plugin to add the "page" to the blog post context dictionary.

```
# Build a context for each post
postContext = {}
postContext['page'] = page
```

You can then use ```{% url page.url %}``` in the templates.

**Comment by blattms** [comment]
Should be resolved duie to the merge of #229
